test/integration/egress.go

```diff
@@ -65,4 +75,13 @@ func (t *egress) run() error {
 }
 
 func (t *egress) teardown() {
+	if !t.Egress {
+		return
+	}
+	if err := client.CoreV1().Services(t.Namespace).Delete("httpbin", &meta_v1.DeleteOptions{}); err != nil {
```

Comment:
yeah not sure what's better - hardcoding file name or service names. both are referenced in the file already.